master/buildbot/wamp/connector.py

```diff
@@ -33,6 +33,8 @@ class MasterService(ApplicationSession, service.AsyncMultiService):
     """
 
     def __init__(self, config):
+        # Cannot use super() here.
+        # We must explicitly call both parent constructors.
```

Comment:
I think that in case of mixins we should call `super().__init__` in their `__init__` even if they don't subclass any classes. This way we don't need workarounds like this. Are both of the base classes not supposed to be used as mixins?